encyctng/static_src/sass/components/_footer.scss

```diff
@@ -0,0 +1,164 @@
+            padding-top: 40px;
+        }
+    }
+
+    &__quick-link {
+        display: flex;
+        align-items: center;
+        color: var(--color--white);
+    }
+
+    &__quick-link-icon {
+        color: var(--color--orange);
+        margin-left: 7px;
+    }
+
+    &__about {
+        display: flex;
+        flex-direction: column;
+        padding-bottom: 40px;
+        border-bottom: 1px solid var(--color--border-dark);
+
+        @include media-query(large) {
+            justify-content: center;
+            border-bottom: none;
+            border-right: 1px solid var(--color--border-dark);
+            padding-right: 80px;
+            padding-bottom: 0;
+        }
+    }
+
+    &__about-link {
+        font-weight: bold;
+        color: var(--color--orange);
+    }
+
+    &__about-intro {
+        @include font-size(introduction);
+        margin-bottom: 20px;
+    }
+
+    &__about-subtitle {
+        @include font-size(meta-large);
+        color: var(--color--light-blue);
+    }
+
+    // Footer bottom section
+    &__bottom {
+        border-top: 1px solid var(--color--border-dark);
+    }
+
+    &__newsletter {
+        position: relative;
+        padding-top: 25px;
+        padding-bottom: 60px;
+        border-top: 1px solid var(--color--border-dark);
+
+        ::before {
+            content: "";
+            position: absolute;
+            top: -2.5px;
+            left: 15px;
+            transform: translateX(-50%);
+            width: 30px;
+            height: 5px;
+            background-color: var(--color--white);
+            border-radius: 3px;
+        }
+
+        @include media-query(large) {
+            border-top: none;
+            padding-bottom: 0;
+        }
+    }
+
+    &__contact {
+        position: relative;
+        padding-top: 25px;
+        padding-bottom: 60px;
+        border-top: 1px solid var(--color--border-dark);
+
+        ::before {
+            content: "";
+            position: absolute;
+            top: -2.5px;
+            left: 15px;
+            transform: translateX(-50%);
+            width: 30px;
+            height: 5px;
+            background-color: var(--color--white);
+            border-radius: 3px;
+        }
+
+        @include media-query(large) {
+            border-top: none;
+            padding-bottom: 0;
+        }
+    }
+
+    &__contact-email {
+        @include font-size(body);
+        color: var(--color--orange);
+        font-weight: bold;
+        margin-bottom: 5px;
+    }
+
+    &__company {
+        padding-top: 30px;
+        border-top: 1px solid var(--color--border-dark);
+
+        @include media-query(large) {
+            border-top: none;
+            padding-left: 80px;
+        }
+    }
+
+    &__company-link {
+        @include font-size(meta-small);
+        color: var(--color--orange);
+        font-weight: bold;
+        margin-right: 20px;
+
+        @include media-query(large) {
+            margin-right: 30px;
+        }
+    }
+}
```

Comment:
Could we use the spacer variables for padding/margins? It looks like we have some consistent spacing between elements which could use variables instead of pixel sizes